tgui/packages/tgui/interfaces/OreSilo.tsx

```diff
@@ -0,0 +1,199 @@
+import { classes } from 'common/react';
+import { capitalize } from 'common/string';
+import { useState } from 'react';
+
+import { useBackend } from '../backend';
+import {
+  Box,
+  Icon,
+  Image,
+  LabeledList,
+  NoticeBox,
+  Section,
+  Stack,
+  Tabs,
+  Tooltip,
+} from '../components';
+import { Window } from '../layouts';
+import { MaterialAccessBar } from './Fabrication/MaterialAccessBar';
+import { Material } from './Fabrication/Types';
+
+interface Machine {
+  ref: string;
+  name: string;
+  icon: string;
+  onHold: boolean;
+  location: string;
+}
+
+interface Log {
+  rawMaterials: string;
+  machineName: string;
+  areaName: string;
+  action: string;
+  amount: number;
+  time: string;
+  noun: string;
+}
+
+type OreSiloData = {
+  SHEET_MATERIAL_AMOUNT: number;
```

Comment:
This should just be cased normally, looks weird like this

Reply:
This parameter is being handled by `/datum/component/material_container` in `ui_static_data` proc, I can't change it, unfortunately.

Reply:
Oh, wait. I'm dumb, huh.